core/src/main/java/io/kestra/core/tasks/flows/Flow.java

```diff
@@ -111,6 +111,12 @@ public Flow.Output run(RunContext runContext) throws Exception {
     public Execution createExecution(RunContext runContext, FlowExecutorInterface flowExecutorInterface) throws Exception {
         RunnerUtils runnerUtils = runContext.getApplicationContext().getBean(RunnerUtils.class);
 
+        Map<String, String> flowVars = (Map<String, String>) runContext.getVariables().get("flow");
+
+        if (flowVars.get("id") == ((Map<?, ?>) runContext.getVariables().get("flow")).get("id") && flowVars.get("namespace") == ((Map<?, ?>) runContext.getVariables().get("flow")).get("namespace")) {
+            throw new IllegalStateException("Recursive flow call detected");
+        }
+
```

Comment:
If we cannot create a revursive flow, no need to check this at runtime.
Moreover it seems that this is always true!

Reply:
Dont we care about possibly existing recursive flow ?

Reply:
I would say no, let's keep it simple for now